kernel/build-kernel.sh

```diff
@@ -195,21 +195,30 @@ setup_kernel() {
 
 	local major_kernel
 	major_kernel=$(get_major_kernel_version "${kernel_version}")
-	kernel_patches=$(find "${patches_path}/${major_kernel}.x" -name '*.patch' -type f)
+	local patches_dir_for_version="${patches_path}/${major_kernel}.x"
+	local kernel_patches=""
+	if [ -d "${patches_dir_for_version}" ]; then
+		kernel_patches=$(find "${patches_dir_for_version}" -name '*.patch' -type f)
+	else
+		info "kernel patches directory does not exit"
 
-	pushd "${kernel_path}" >>/dev/null
+	fi
+
+	[ -n "${arch_target}" ] || arch_target="$(uname -m)"
+	arch_target=$(arch_to_kernel "${arch_target}")
+	(
+	cd "${kernel_path}" || exit 1
```

Comment:
heh, be nice if you indented for the `( )` bit ;-)

Reply:
done :)